src/js/view/preview.js

```diff
@@ -29,6 +38,26 @@ Object.assign(Preview.prototype, {
         style(this.el, {
             backgroundImage: backgroundImage
         });
+       
+        if (this.hasZoomThumbnail) {
+            this.enableZoomThumbnail();     
+        }
+    },
+    enableZoomThumbnail: function() {
+        if (this.model.get('isFloating')) {
+            return;
+        }
+        
+        this.zoomThumbnailTimeout = setTimeout(() => {
+            this.el.classList.add('jw-ab-zoom-thumbnail');
+            this.el.style.transformOrigin = this.zoomOriginX + ' ' + this.zoomOriginY;
+        }, 2000);
+    },
+    pauseZoomThumbnail: function() {
+        this.el.style.animationPlayState = this.model.get('viewable') ? 'running' : 'paused';
```

Comment:
`clearTimeout(this.zoomThumbnailTimeout);` here incase `pauseZoomThumbnail` runs before the `enableZoomThumbnail` timeout.